### DISCUSSION TITLE
IMage build fails when i add ftp extension

### DISCUSSION THREAD

**Discussion by thedinz** [docker-library/wordpress#880] [General]
Am i doing it wrong?

I simple add ftp \ under exif \

and it fails cannot create ftp.lo

**Comment by tianon** [comment]
I tested with the following `Dockerfile` and wasn't able to reproduce an issue: :sweat_smile:

```dockerfile
FROM wordpress
RUN docker-php-ext-install ftp
```

**Reply by thedinz** [reply]
that is very odd, i was able to fix it but i need to also add libssl-dev \

Odd that you did not

It was specifically this image if that matters: https://github.com/docker-library/wordpress/tree/06f28e291877002359e1075e4260123de5403551/latest/php8.1

**Reply by tianon** [reply]
If you just need the FTP extension, you don't have to (and probably don't want to) rebuild the full Wordpress or PHP image.

I guess you do need `libssl-dev` as well (even with my example) if you want `ftp_ssl_connect`: https://github.com/docker-library/php/issues/1488#issuecomment-1906738878 :innocent:
